### DISCUSSION TITLE
前端小白请教如何解决该问题：, 可能是我用的es6 语法？ 但是我没有办法该这个源码，如何解决啊

### DISCUSSION THREAD

**Discussion by lucasjinreal** [ant-design/ant-design#43563] [Q&A]
### Reproduction link

[https://baidu.com](https://baidu.com)

### Steps to reproduce

前端小白请教如何解决该问题：

node_modules\antd\es\input\Password.js:9
import EyeInvisibleOutlined from "@ant-design/icons/es/icons/EyeInvisibleOutlined";
^^^^^^
相关版本：

"@ant-design/icons": "^5.1.4",
    "@ant-design/pro-components": "^2.6.4",
    "@dqbd/tiktoken": "^1.0.2",
    "@fortawesome/fontawesome-svg-core": "^6.4.0",
    "@fortawesome/free-solid-svg-icons": "^6.4.0",
    "@fortawesome/react-fontawesome": "^0.2.0",
    "@tabler/icons-react": "^2.9.0",
    "antd": "^5.6.3",
可能是我用的es6 语法？ 但是我没有办法该这个源码，如何解决啊

### What is expected?

前端小白请教如何解决该问题：

node_modules\antd\es\input\Password.js:9
import EyeInvisibleOutlined from "@ant-design/icons/es/icons/EyeInvisibleOutlined";
^^^^^^
相关版本：

"@ant-design/icons": "^5.1.4",
    "@ant-design/pro-components": "^2.6.4",
    "@dqbd/tiktoken": "^1.0.2",
    "@fortawesome/fontawesome-svg-core": "^6.4.0",
    "@fortawesome/free-solid-svg-icons": "^6.4.0",
    "@fortawesome/react-fontawesome": "^0.2.0",
    "@tabler/icons-react": "^2.9.0",
    "antd": "^5.6.3",
可能是我用的es6 语法？ 但是我没有办法该这个源码，如何解决啊

### What is actually happening?

前端小白请教如何解决该问题：

node_modules\antd\es\input\Password.js:9
import EyeInvisibleOutlined from "@ant-design/icons/es/icons/EyeInvisibleOutlined";
^^^^^^
相关版本：

"@ant-design/icons": "^5.1.4",
    "@ant-design/pro-components": "^2.6.4",
    "@dqbd/tiktoken": "^1.0.2",
    "@fortawesome/fontawesome-svg-core": "^6.4.0",
    "@fortawesome/free-solid-svg-icons": "^6.4.0",
    "@fortawesome/react-fontawesome": "^0.2.0",
    "@tabler/icons-react": "^2.9.0",
    "antd": "^5.6.3",
可能是我用的es6 语法？ 但是我没有办法该这个源码，如何解决啊

| Environment | Info |
| --- | --- |
| antd | 5.7.0 |
| React | 5.4 |
| System | windows |
| Browser | chrome |

---
前端小白请教如何解决该问题：

node_modules\antd\es\input\Password.js:9
import EyeInvisibleOutlined from "@ant-design/icons/es/icons/EyeInvisibleOutlined";
^^^^^^
相关版本：

"@ant-design/icons": "^5.1.4",
    "@ant-design/pro-components": "^2.6.4",
    "@dqbd/tiktoken": "^1.0.2",
    "@fortawesome/fontawesome-svg-core": "^6.4.0",
    "@fortawesome/free-solid-svg-icons": "^6.4.0",
    "@fortawesome/react-fontawesome": "^0.2.0",
    "@tabler/icons-react": "^2.9.0",
    "antd": "^5.6.3",
可能是我用的es6 语法？ 但是我没有办法该这个源码，如何解决啊

**Comment by kiner-tang** [comment]
直接这样引入：
```tsx
import { EyeInvisibleOutlined } from "@ant-design/icons"
```

**Reply by lucasjinreal** [reply]
你好，感谢回答！ @kiner-tang  但是这个是antd里面的代码。。。。 我自己没有import 这玩意，问题是

**Comment by zombieJ** [comment]
对 webpack 不熟悉的话可以直接找一些现成的框架来用，比如 https://umijs.org/

**Reply by lucasjinreal** [reply]
@zombieJ  你好，感谢回答！   但是这个是antd里面的代码。。。。 我自己没有import 这玩意，问题是
